### PR TITLE
List the approved mentors of a program in the public view

### LINKED ISSUE #43: Implementing the view for showing all the mentors in a program
**Is your feature request related to a problem? Please describe.**
- Once the program reached the lifecycle state of MENTEE_APPLICATION_PERIOD and in the MENTEE_VERIFICATION_PERIOD a mentee can apply for mentors. So my task is to list down all the mentors that available in that particular program.

**Describe the solution you'd like**
- Create the view for list all the mentors

**Additional context**
- For more information: https://sef.discourse.group/t/implementing-the-view-for-showing-all-the-mentors-in-a-program/198

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #43 

## Goals

- As a mentee, during the mentor verification period mentee should be able to apply for mentors 

## Approach

- Changed the directory structure.
- Created the request mentors view 
## Screenshots

image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/scenes/Home/scenes/RequestMentors/index.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <div className={mainStyles.container}>
+          <Title>Request Mentors</Title>
+          <Spin tip="Loading..." spinning={isLoading}>
+            <List
+              itemLayout="horizontal"
+              size="large"
+              pagination={{
+                onChange: (page) => {
+                  console.log(page);
+                },
```
Comment (jayasanka-sack) [inline src/scenes/Home/scenes/RequestMentors/index.tsx]: Remove this event listener

File: src/scenes/Home/scenes/RequestMentors/index.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                pageSize: 8,
```
Comment (jayasanka-sack) [inline src/scenes/Home/scenes/RequestMentors/index.tsx:55]: The page size is hard coded here

File: src/scenes/Home/scenes/RequestMentors/index.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        if (result.status == 200) {
+          setIsLoading(false);
+          setMentors(result.data);
+        } else {
+          throw new Error();
+        }
+      })
+      .catch(() => {
+        setIsLoading(false);
+        notification.warning({
```
Comment (jayasanka-sack) [inline src/scenes/Home/scenes/RequestMentors/index.tsx]:
```suggestion
        notification.error({
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/scenes/Home/scenes/RequestMentors/index.tsx (hunk 2 above)
Comment (anjula-sack) [reply src/scenes/Home/scenes/RequestMentors/index.tsx:55]: pageSIze means no of items per page

File: src/scenes/Home/scenes/RequestMentors/index.tsx (hunk 1 above)
Comment (anjula-sack) [reply src/scenes/Home/scenes/RequestMentors/index.tsx]: ok

File: src/scenes/Home/scenes/RequestMentors/index.tsx (hunk 3 above)
Comment (anjula-sack) [reply src/scenes/Home/scenes/RequestMentors/index.tsx]: ok

**Review by jayasanka-sack — APPROVED** [review]
Thanks @anjula-sack !

**Pull request merged**